tinyengine/core.py

```diff
@@ -53,65 +53,74 @@ def __str__(self):
 
     @property
     def position(self):
-        return (self.x, self.y)
+        return self.x, self.y
 
     @property
     def up(self):
         self.y -= 1
-        return Vector(self.x, self.y)
+        return self.position
 
     @property
     def down(self):
         self.y += 1
-        return Vector(self.x, self.y)
+        return self.position
 
     @property
     def right(self):
         self.x += 1
-        return Vector(self.x, self.y)
+        return self.position
 
     @property
     def left(self):
         self.x -= 1
-        return Vector(self.x, self.y)
+        return self.position
 
     @property
     def zero(self):
-        return Vector(0, 0)
+        self.x = 0
+        self.y = 0
+        return self.position
 
 
-class RGB():
+class RGB:
     def __init__(self, x=0, y=0, z=0):
```

Comment:
The engine is planned to be a 2D engine, so the z axis should be removed.

Reply:
It's RGB, yo!